src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml

```diff
@@ -137,7 +137,7 @@
                          AutomationProperties.Name="{x:Static Properties:Resources.tbTeamProjectSearchAutomationPropertiesName}"
                          FontSize="{DynamicResource StandardTextSize}"/>
                 <fabric:FabricIconControl HorizontalAlignment="Right" Margin="5,0,5,0" 
-                                                  GlyphName="Search" GlyphSize="Custom" FontSize="11" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=IconBrush}"/>
+                                                  GlyphName="Search" GlyphSize="Custom" FontSize="11" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=IconBrush}" ShowInControlView="False"/>
```

Comment:
![screenshot of search in bug filing setup](https://user-images.githubusercontent.com/7775527/64988627-bd9b5d00-d880-11e9-9375-652d38662635.png)

Reply:
I'm not sure this particular icon is non-decorative because I think it's adding new information for a visual user; without it, I'd have assumed I'd need an exact team name, but with it, I'd assume I could search based on a partial name.

Maybe this could be solved by adding placeholder text to the search box itself, like the HierarchyControl and PropertyInfoControl cases?

Reply:
@dbjorge agreed & moved back to control view for now. I agree placeholder text would be nice and don't think it belongs in this PR of `ShowInControlView` changes